MovieApp/MovieApp/App/Features/MovieBrowser/Components/MovieCellHPv1/MovieCell.swift

```diff
@@ -1,34 +1,35 @@
 import UIKit
 import Kingfisher
 
-class MovieCell: UICollectionViewCell {
+class MovieCell: UITableViewCell {
     
     static let reuseIdentifier = String(describing: MovieCell.self)
     
-    let descriptionFontSize: CGFloat = 14
+    var movieId: Int!
+    
     let offset: CGFloat = 8
     let movieImageWidth: CGFloat = 97
-    let movieTitleViewHeight: CGFloat = 20
     
     var movieTitle: UILabel!
     var movieDescription: UILabel!
     var movieImageView: UIImageView!
-    var contentContainer: UIView!
-    
-    override init(frame: CGRect) {
-        super.init(frame: frame)
         
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
         buildViews()
     }
     
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
     
-    func populate(withMovie movie: MovieViewModel) {
+    func populate(with movie: MovieViewModel) {
+        movieId = movie.identifier
```

Comment:
no need for this. Cell doesn't need to have a variable to save ids. You can can get that from UITableViewDataSource didSelectRowAt

Reply:
[Removed](https://github.com/lihter/Movie-App/pull/41/commits/3383b62f0e45eb7e1f5ba5b650a01beaa73387a1#diff-59acdf9fea7a34d1960015778fa4fae8eba6ce40a9b91286052e7d63a97de079L28)

Reply:
[LINK MADAFAKA](https://github.com/lihter/Movie-App/pull/41/commits/3383b62f0e45eb7e1f5ba5b650a01beaa73387a1#diff-6a7e0c3cc088b3a980b8a92c08ad5d190f909d67f14aa13f150bdc992ab52302R145)